test/commands-tests/init.js

```diff
@@ -0,0 +1,48 @@
+const chai = require('chai');
+const chaiFiles = require('chai-files');
+const assert = chai.assert;
+const execute = require('../../utils.js').execute;
+const cleanUp = require("../utils").cleanUp
+const fs = require('fs-extra')
+
+let executeOptions = {
+	cwd: process.cwd() + "./test/commands-tests/initTests"
+};
+let expect = chai.expect;
+let file = chaiFiles.file;
+let dir = chaiFiles.dir;
+
+chai.use(chaiFiles);
+
+
+describe('Aeproject Init command', () => {
+	before(async () => {
+		fs.ensureDirSync('./test/commands-tests/initTests')
+	})
+
+	it('Should init project successfully', async () => {
+		let result = await execute("init", [], executeOptions)
+
+		expect(file(executeOptions.cwd + '/package.json')).to.exist;
```

Comment:
All hardcoded strings should be in constnts file. Also the current standard is to use string interpolation `${executeOptions.cwd}/package.json` would be the equivalent of what you've written